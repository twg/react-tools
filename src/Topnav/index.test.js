/* eslint-env jest */
import React from 'react'
import { TopNav } from './'
import renderer from 'react-test-renderer'

describe('TopNav', () => {
  it('matches snapshot', () => {
    let component = renderer.create(
      <div
        style={{
          height: '300px',
          maxWidth: '740px',
          width: '100%',
          border: '1px solid grey'
        }}
      >
        <TopNav
          left={
            <img
              src="https://twg.io/954c24bb4bbfe909730f85f6c55c8e9e.svg"
              width="175px"
            />
          }
          center={<h4>Page Title</h4>}
          links={['item', 'item2', 'item2', 'item2', 'item2']}
          right={
            <img
              class="avatar js-action-profile-avatar"
              src="https://pbs.twimg.com/profile_images/608742547272290304/vl4pXqqG_bigger.png"
              style={{ borderRadius: '50%', height: '58px' }}
              alt=""
            />
          }
        />
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
