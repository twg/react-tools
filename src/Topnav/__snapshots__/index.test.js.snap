// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopNav matches snapshot 1`] = `
<div
  style={
    Object {
      "border": "1px solid grey",
      "height": "300px",
      "maxWidth": "740px",
      "width": "100%",
    }
  }
>
  <div
    className="topnav"
    style={Object {}}
  >
    <div
      className="left"
    >
      <button
        className="mobileButton"
        onClick={[Function]}
      >
        â˜°
      </button>
      <img
        src="https://twg.io/954c24bb4bbfe909730f85f6c55c8e9e.svg"
        width="175px"
      />
    </div>
    <div
      className="center"
    >
      <h4>
        Page Title
      </h4>
    </div>
    <div
      className="right"
    >
      <ul
        style={Object {}}
      >
        <li>
          item
        </li>
        <li>
          item2
        </li>
        <li>
          item2
        </li>
        <li>
          item2
        </li>
        <li>
          item2
        </li>
      </ul>
      <div
        style={
          Object {
            "display": "inherit",
          }
        }
      >
        <img
          alt=""
          class="avatar js-action-profile-avatar"
          src="https://pbs.twimg.com/profile_images/608742547272290304/vl4pXqqG_bigger.png"
          style={
            Object {
              "borderRadius": "50%",
              "height": "58px",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
